### ISSUE TITLE
Shutdown AFD ETL and DB server on VZ Lightsail Server

### ISSUE THREAD

**Issue by patrickm02L** [cityofaustin/atd-data-tech#8826]
As of April 1, 2022, we're receiving AFD data to our S3 bucket which gets added to the VZDB. We can shut down the Lightsail server that was previously collecting AFD data.

- [x] Inventory services still in place
- [ ] Move everything to on-prem services excluding AFD data
- [x] Report back feasibility of shutting down server
- [ ] ~~Let Gus know to remove Frank's account from the distribution list~~

**Comment by frankhereford** [comment]
**TLDR:** There's the coordination tool that on there, and we can move that on-prem pretty easy. I'd call that task a `3`. Everything else is good to depricate. Thank you for asking!

The lightsail server is currently offering up:

* The VZ Coordination tool, which has the following stack components:
  * postgres postgis server for GIS data
  * backend endpoint served via `apache` with perl endpoint for ajax requests from the app
  * martin server to expose the postgres data as vector tiles
  * actual server for the compiled JS client app
  * a raster tile servers for base maps
  * haproxy for SSL termination and reverse proxy capabilities
 * A old copy of the crash diagram tool that is long superseded by our work with Adrien. Never going to need that again.

And of course, the AFD ETL and DB server for said  data. This should be out of service pretty quick thanks to Mateo's awesome work integrating that into the VZ suite.

**Comment by mateoclarke** [comment]
I would say migrating the VZ Coordination tool should be pretty low priority and treated as a separate issue. I believe the motivation for this issue was strictly related to the AFD data. Is that true @patrickm02L? I would propose we re-name this issue "Shutdown AFD ETL and DB server on VZ Lightsail Server"

The only benefit I see to migrating the VZ Coordination tool is minor cost savings.

**Comment by patrickm02L** [comment]
@mateoclarke the motivation was in relation to the AFD data yes. In talking with Frank he realized the VZ Coordination tool was also on the Lightsail server. I wasn't sure if we were going to shut down the server completely, so I asked him to do an inventory of what was on Lightsail and report back.

If leaving the Coordination tool on Lightsail works, then we can ignore the migration. Begs the question if should migrate at all? @mateoclarke if you think we should, I'll make a new issue.

**Comment by patrickm02L** [comment]
Chatted with Xavier 10/4/22, he said that all his queries for AFD are now pointing to vzdb and no longer to the Lightsail server. He said it's good to retire now.

**Comment by frankhereford** [comment]
Thanks Patrick for that info. I still need to get a thumbs up from Caitlin to actually do this, but I should be able to get that from her this week.

**Comment by frankhereford** [comment]
@patrickm02L - I wrote and heard back from Caitlin, I believe you were CC'd on that. When you're available next week, let's sync and game plan if that's suitable for you.

**Comment by frankhereford** [comment]
Touching back on this - I wrote Caitlin on Oct 17, and I have not heard back. @patrickm02L, I think one of us should reach out again - do you have a preference on who does that?

**Comment by patrickm02L** [comment]
This Issue still remains because we're waiting to hear back as to whether or not Caitlin wants to continue with #10088. Until we know this Issue is in purgatory.

**Comment by frankhereford** [comment]
Should I replace the SSL certificate on it? https://capmetro.atxvisionzero.xyz/ is expired. It'd be a 1.

**Comment by patrickm02L** [comment]
Talked about this in Dev grooming 12.21.22

No. Once we get sign off from Caitlin about the [AGOL map](https://austin.maps.arcgis.com/apps/webappviewer/index.html?id=f150a80a0c85407892ce850fea3346c9) that Andrew put together. We'll shut this down.

**Comment by patrickm02L** [comment]
We've delivered the AGOL map to Caitlin - considering that the action that we needed to take to make the Lightsail server obsolete. 

@frankhereford would you please shut down Lightsail, and close out this Issue.

**Comment by frankhereford** [comment]
The disk of this machine was snapshotted and transferred to a snapshot stored in our EC2 EBS store. It's 80 gigs, so it will burn about a $1 a month sitting there, so I am inclined to keep it there for 6 months or so, just on the very off-chance we need to restore it.

The lightsail server has been retired, and I'll retire the DNS entries over the weekend. They are personal domains, kept over from the VZ days.

**Comment by patrickm02L** [comment]
Thanks Frank. I think keeping it there for 6-months makes sense.
